README.md

```diff
@@ -213,26 +278,26 @@ As already discussed, employees can get restricted stock, stock options, or RSUs
+- Generally, restricted stock is taxed as ordinary income *[when it vests](http://www.investopedia.com/articles/tax/09/restricted-stock-tax.asp?performancelayout=true).* If the stock is in a startup with low value, this may not result in a high tax. If it’s been years since the stock was first granted and the company is now worth a lot, the taxes owed could be quite significant.
+- 🔹 The Internal Revenue Code offers an alternative that will protect you from this high potential tax. It’s called a [**Section 83(b) election**](http://acceleratedvesting.com/what-is-an-83b-election-and-when-do-i-make-it-part-1-with-graphic/), and ensures that you’ll be taxed on the receipt of the “property” (the stock), rather than at the time the stock vests. Basically, with a Section 83(b) election, you’re telling the IRS you want to pay taxes early, and this can potentially reduce your tax significantly: If the shares go up in value, the taxes owed on vesting might be far greater than the taxes owed at the time of receipt.
+    - An 83(b) election isn’t guaranteed to reduce your taxes, of course. For example, the value of the stock may not increase. And if you leave the company before you vest, you *don’t* get the taxes you’ve paid back.
     - ❗ You must file the 83(b) election yourself with the IRS [**within 30 days**](https://www.irs.gov/irb/2012-28_IRB/ar12.html) of the grant or exercise, or the opportunity is irrevocably lost.
     - 🔸 Section 83(b) elections cannot be made on the receipt of a stock option. They can only be made on the receipt of actual shares of stock.
-    - If you receive an immediately exercisable stock option (meaning, an option that is early exercisable, when it is not vested), and you exercise the option before the option vests, you can make an 83(b) election on your receipt of the shares on exercise.
-    - Section 83(b) elections do not apply to vested shares; the election only applies to stock that is not yet vested. Thus, if you receive options that are *not* early exercisable, which you cannot exercise until vested, then an 83(b) election would not apply.
+    - If you receive an early exercisable stock option (when you don’t have to wait for the the stock to vest), you can make an 83(b) election on receipt of the shares on exercise.
+    - Section 83(b) elections do not apply to vested shares; the election only applies to stock that is not yet vested. Thus, if you receive options that are *not* early exercisable (meaning you have to wait until they vest to exercise), an 83(b) election would not apply.
     - 🔹 Founders and very early employees will almost always want to do an 83(b) election upon the receipt of unvested shares, since the stock value is probably low. If the value is really low, and the taxes owed are not that great, you can make the election without having to pay much tax and start your capital gains holding period on the shares.
 
 ### 409A valuations
```

Comment:
This is the best article I have read on the subject, despite an somewhat incendiary title: https://www.nytimes.com/2017/03/08/business/dealbook/valuation-shell-game-silicon-valleys-dirty-secret.html

Reply:
✔️